src/dotnet/APIView/APIViewWeb/Startup.cs

```diff
@@ -228,18 +233,18 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<IAuthorizationHandler, OrganizationRequirementHandler>();
             services.AddSingleton<IAuthorizationHandler, CommentOwnerRequirementHandler>();
             services.AddSingleton<IAuthorizationHandler, ReviewOwnerRequirementHandler>();
-            services.AddSingleton<IAuthorizationHandler, RevisionOwnerRequirementHandler>();
+            services.AddSingleton<IAuthorizationHandler, APIRevisionOwnerRequirementHandler>();
             services.AddSingleton<IAuthorizationHandler, ApproverRequirementHandler>();
             services.AddSingleton<IAuthorizationHandler, ResolverRequirementHandler>();
-            services.AddSingleton<IAuthorizationHandler, AutoReviewModifierRequirementHandler>();
-            services.AddSingleton<IAuthorizationHandler, UsageSampleOwnerRequirementHandler>();
+            services.AddSingleton<IAuthorizationHandler, AutoAPIRevisionModifierRequirementHandler>();
+            services.AddSingleton<IAuthorizationHandler, SamplesRevisionOwnerRequirementHandler>();
             services.AddSingleton<CosmosClient>(x =>
             {
                 return new CosmosClient(Configuration["Cosmos:ConnectionString"]);
             });
 
             services.AddHostedService<ReviewBackgroundHostedService>();
-            services.AddHostedService<PullRequestBackgroundHostedService>();
+            //services.AddHostedService<PullRequestBackgroundHostedService>();
```

Comment:
Why is this commented? Pull requested background task closes PR revision 30 days after a PR is closed.